bitflags 1.3.0 breaks nix

https://github.com/bitflags/bitflags/pull/187 breaks `nix` because it does not respect existing `repr` attributes.

```
error[E0692]: transparent struct cannot have other repr hints
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/nix-0.22.0/src/sys/statvfs.rs:15:12
   |
13 | / libc_bitflags!(
14 | |     /// File system mount Flags
15 | |     #[repr(C)]
   | |            ^
16 | |     #[derive(Default)]
...  |
52 | |     }
53 | | );
   | |__^
   |
   = note: this error originates in the macro `::bitflags::bitflags` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0692`.
```

cc @flier @KodrAus

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -342,7 +342,6 @@ macro_rules! bitflags {
         $($t:tt)*
     ) => {
         $(#[$outer])*
-        #[repr(transparent)]
         #[derive(Copy, PartialEq, Eq, Clone, PartialOrd, Ord, Hash)]
         $vis struct $BitFlags {
             bits: $T,

